discord/interactions.py

```diff
@@ -303,10 +303,28 @@ async def original_message(self) -> InteractionMessage:
         )
         state = _InteractionMessageState(self, self._state)
         message = InteractionMessage(state=state, channel=channel, data=data)  # type: ignore
-        self._original_message = message
+        self._original_response = message
         return message
 
-    async def edit_original_message(
+    async def original_message(self, **kwargs):
```

Comment:
Very unnecessary, `original_response` doesn't take in any arguments

Reply:
I removed `**kwargs`.